tsconfig.eslint.json

```diff
@@ -2,11 +2,8 @@
   "extends": "./tsconfig.json",
   "include": [
     "index.ts",
+    "bin/**/*.ts",
     "lib/**/*.ts",
-    "test/**/*.ts",
```

Comment:
Does this mean tests won't get linted anymore?

Reply:
The diff makes it look like the line was removed, but it is actually still there on line 7. I had to check it myself, too.